### ISSUE TITLE
TTSR model key error when inference .

### ISSUE THREAD

**Issue by ThorPham** [open-mmlab/mmediting#613]
Hi,

I got the error that: KeyError: 'ref_path' when I execute the following command:
python demo/restoration_demo.py configs/restorers/ttsr/ttsr-gan_x4_c64b16_g1_500k_CUFED.py  checkpoint/ttsr/ttsr-gan_x4_c64b16_g1_500k_CUFED_20210626-2ab28ca0.pth  /home/thorpham/Documents/challenge/test/img_5.jpg  outputs/demo_out_baboon.png --imshow 

>   File "demo/restoration_demo.py", line 46, in <module>
    main()
  File "demo/restoration_demo.py", line 37, in main
    output = restoration_inference(model, args.img_path)
  File "/home/thorpham/Documents/challenge/mmediting/mmedit/apis/restoration_inference.py", line 36, in restoration_inference
    data = test_pipeline(data)
  File "/home/thorpham/Documents/challenge/mmediting/mmedit/datasets/pipelines/compose.py", line 42, in __call__
    data = t(data)
  File "/home/thorpham/Documents/challenge/mmediting/mmedit/datasets/pipelines/loading.py", line 61, in __call__
    filepath = str(results[f'{self.key}_path'])
KeyError: 'ref_path'

How to fix it . Thank you

**Comment by ckkelvinchan** [comment]
@Yshuo-Li

**Comment by Yshuo-Li** [comment]
I'm sorry for the previous code doesn't support inference of RefSR methods such as TTSR, and I have fixed it in #716 

@ThorPham now you can download the PR branch by 
```
git fetch origin pull/716/head:pr716
git checkout pr716
```
where `origin` is the remote name of our repo (can be set by `git remote add origin https://github.com/open-mmlab/mmediting.git`).
Then you are able to run inference of TTSR by
```
python demo/restoration_demo.py configs/restorers/ttsr/ttsr-gan_x4_c64b16_g1_500k_CUFED.py checkpoint/ttsr/ttsr-gan_x4_c64b16_g1_500k_CUFED_20210626-2ab28ca0.pth /home/thorpham/Documents/challenge/test/img_5.jpg outputs/demo_out_baboon.png --imshow --ref_path [REF_PATH]
```
where `REF_PATH` is the file path of reference image.

Besides, once #716 is merged into master, you can run inference of TTSR directly.

**Comment by Yshuo-Li** [comment]
#716 have been merged into master, now you can run inference of TTSR by
```
python demo/restoration_demo.py configs/restorers/ttsr/ttsr-gan_x4_c64b16_g1_500k_CUFED.py checkpoint/ttsr/ttsr-gan_x4_c64b16_g1_500k_CUFED_20210626-2ab28ca0.pth /home/thorpham/Documents/challenge/test/img_5.jpg outputs/demo_out_baboon.png --imshow --ref_path [REF_PATH]
```
where `REF_PATH` is the file path of reference image.

**Comment by zengyh1900** [comment]
Closing this issue since the bug has been fixed in a merged pull request.
Please feel free to reopen it if it has not been fixed
